prov/efa/src/rxr/rxr_init.c

```diff
@@ -578,6 +564,15 @@ static int rxr_getinfo(uint32_t version, const char *node,
 			goto free_info;
 
 		ofi_alter_info(util_info, hints, version);
+
+		/* If application asked for FI_REMOTE_COMM but not FI_LOCAL_COMM, it
+		 * does not want to use shm. In this case, we honor the request by
+		 * unsetting the FI_LOCAL_COMM flag in info. This way rxr_endpoint()
+		 * should disable shm transfer for the endpoint
+		 */
+		if (hints && hints->caps & FI_REMOTE_COMM && !(hints->caps & FI_LOCAL_COMM))
+			util_info->caps &= ~FI_LOCAL_COMM;
```

Comment:
This might fit right in with the things we do in `rxr_info_to_rxr()`; but this is fine too given we want to rip out the rxr_info_to_*() logic.

Reply:
Thanks, will leave it here then.